xls/dslx/stdlib/std.x

```diff
@@ -396,6 +396,41 @@ fn round_up_to_nearest_test() {
     assert_eq(u32:4, round_up_to_nearest(u32:4, u32:2));
 }
 
+// Returns `x` rounded up to `y`, where `y` is a positive power of 2
```

Comment:
can you detail a little bit more what "rounded up to `y`" implies for each cases (when x < y, when x == y, when x > y) from reading the test I'm not sure I have a good understanding of it

Reply:
Sorry for the confusion. The function should do the same as the existing std::round_up_to_nearest(), but generate much simpler logic.

Reply:
Can you note this in the comment? i.e. in the second line

// This functionality is the same as `round_up_to_nearest` but more optimized when `y` is a known power of 2.